doc/wiki/running-ab-experiments.md

```diff
@@ -3,39 +3,59 @@
 
-## Active patient experiment:
-- experiment will be started by running a script either manually or by scheduling it
-- the job can be run like so: `ExperimentControlServerice.start_current_patient_experiment(name: "experiment name", days_til_start: 3, days_til_end: 33, percentage_of_patients: 100)`
-- the `percentage_of_patients` argument is optional and defaults to 100
-- be sure that `days_til_start` leaves enough time to send any reminders that need to be sent before the appointment. For example, if the experiment has a treatment that involves sending reminders three days before the experiment begins, set `days_til_start` to at least four.
-- the job will select patients and assign all eligible patients to treatment groups at random
-- it will create all notifications for experiment patients, based on the date of their next appointment and the information in their treatment group's reminder templates. These will be marked as "pending" and will not be sent at this time.
-- Then the messages will then be scheduled for delivery
+## Current patient experiment:
+- current patient experiments can start sending notifications before the experiment start date. The start date and end date truly mean "selection start date" and "selection end date". Patients are selected for the experiment if they have appointments during the date range. But because reminder templates can be configured to send notifications before or after appointments, it's possible for notifications to be sent before the start date and after the end date. For the experiment to run correctly, the experiment must be created and added to scheduler before notifications should start sending, not just before the experiment start_date.
+- experiments must first be created according to [experiment setup](#experiment-setup).
+- experiments will be run by adding the following to `config/schedule.rb`:
+  - `runner "Experimentation::Runner.start_current_patient_experiment(name: 'experiment name')"`
+  - this also accepts an optional `percentage_of_patients` argument that defaults to 100
+- the first time it is run by the scheduler, the job will:
+  - select eligible patients
+  - assign them randomly to treatment groups
+  - create all notifications for experiment patients, based on the date of their next appointment and the information in their treatment group's reminder templates. These will be marked as "pending" and will not be sent by this job.
+  - change the experiment state from "new" to "running"
+- those messages will then be scheduled for delivery each day as described [here](notification-scheduling).
+- despite the fact that patient selection and notification creation is all done during the first time it runs, this experiment runner can safely be left on the scheduler. On subsequent runs, it will do nothing until the experiment end date has passed, at which point it will change the experiment state to "complete".
```

Comment:
Not sure we need this level of detail -- I'd rather folks go look at the code and the specs to see how the implementation works.  These kinds of docs will be out of date within a week I'd wager.